modules/aioseop_opengraph.php

```diff
@@ -1342,6 +1342,7 @@ function add_meta() {
 
 			if ( $type === 'article' && ! empty( $post ) && is_singular() ) {
 				if ( ! empty( $this->options['aiosp_opengraph_gen_tags'] ) ) {
+					$tag = '';

```

Comment:
This will need to be moved higher up in the function.

1) it could potentually overwrite a variable above https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/03ea3f9130ccfa95dc3d9b96b8be7c696d0d195a/modules/aioseop_opengraph.php#L1274
2) and there would still be an undefined error below with https://github.com/semperfiwebdesign/all-in-one-seo-pack/blob/03ea3f9130ccfa95dc3d9b96b8be7c696d0d195a/modules/aioseop_opengraph.php#L1371

You might have to move it to the top of the function where all the other variables are being set.

Reply:
You are right; good catch!